submissions/lalalilulu/dom-api-menu/script.js

```diff
@@ -0,0 +1,237 @@
+        image: "./images/mask/photo_5.jpg"
+    },
+    {   type: "Маски, сыворотки",
+        title: "Matrix Biolage Fiberstrong Deep Treatment Pack",
+        description: "Маска-концентрат для укрепления ломких, ослабленных волос",
+        price: 850,
+        image: "./images/mask/photo_6.jpg"
+    },
+    {   type: "Стайлинг",
+        title: "John Frieda Volume Lift Thickening",
+        description: "Мусс для уплотнения и придания объема волос",
+        price: 723,
+        image: "./images/styling/photo_1.jpg"
+    },
+    {   type: "Стайлинг",
+        title: "Keune Style Heat Protect Instant Blowout N°37",
+        description: "Термозащитный спрей для быстрой укладки",
+        price: 2118,
+        image: "./images/styling/photo_2.jpg"
+    },
+    {   type: "Стайлинг",
+        title: "Keune Style Straight Cream N°57",
+        description: "Выпрямляющий крем для волос",
+        price: 1686,
+        image: "./images/styling/photo_3.jpg"
+    },
+    {   type: "Стайлинг",
+        title: "John Frieda Volume Lift Root Booster",
+        description: "Спрей для прикорневого объема волос с термозащитным действием",
+        price: 723,
+        image: "./images/styling/photo_4.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "L'Oreal Excellence",
+        description: "Краска для волос",
+        price: 249,
+        image: "./images/coloration/photo_1.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "Schwarzkopf Got2b Bright/Pastel Набор для тонирования волос",
+        description: "Набор для тонирования волос",
+        price: 299,
+        image: "./images/coloration/photo_2.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "L'Oreal Magic Retouch",
+        description: "Краска для волос",
+        price: 204,
+        image: "./images/coloration/photo_3.jpg"
+    },
+    {   type: "Окрашивание",
+        title: "L'Oreal Colorista Spray",
+        description: "Тонирующий спрей для волос",
+        price: 299,
+        image: "./images/coloration/photo_4.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Crystal Clear",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_1.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Pretzel Brown",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_2.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Wrapstar Hair-Rings Snake It Off",
+        description: "Резинка с лентой",
+        price: 1092,
+        image: "./images/accessories/photo_3.jpg"
+    },
+    {   type: "Аксессуары",
+        title: "Invisibobble Original Hair-Rings Pinkerbell",
+        description: "Резинка-браслет для волос",
+        price: 503,
+        image: "./images/accessories/photo_4.jpg"
+    }];
+
+function removeElementsIfExist(elementsList) {
```

Comment:
No need to remove elements and create new ones. Just use `element.innerHTML`, `element.insertAdjacentHTML` for inserting string content into the DOM node or `element.insertAdjacentElement` or `element.append` for inserting DOM nodes. 
It would help you to decrease DOM operations which cost a lot.